references/gcp-sa-auth-shared-flow/README.md

```diff
@@ -0,0 +1,36 @@
+# GCP Service Account Authentication - Shared Flow
+
+This shared flow can be used obtain access tokens for Google Cloud service accounts. Access tokens are cached in the default environment cache resource for 10min.
+
+## Usage instructions
+
+1. Create a service account in Google Cloud and assign it the necessary roles. See GCP [docs](https://cloud.google.com/iam/docs/creating-managing-service-accounts).
+2. Create and download a json key for the service account. See GCP [docs](https://cloud.google.com/iam/docs/creating-managing-service-account-keys).
+3. In your Apigee flow, make sure you have the `private.gcp.service_account.key` variable set. It should hold the full json key for the service account. You can do this by e.g. storing it as a value in an encrypted KVM.
```

Comment:
The recommendation is to store this value in an encrypted KVM and populate a private variable using `KeyValueMapOperations` policy during runtime.